src/finn/transformation/fpgadataflow/insert_hook.py

```diff
@@ -37,7 +37,7 @@
 
 
 def _is_hook_node(node):
-    if node.op_type in ["checksum"]:
+    if node.op_type in ["CheckSum"]:
```

Comment:
If this is the only 'hook' type node, then we could do the check like: `if node.op_type == "CheckSum"`. 
Perhaps in the future, if we plan to extend the types of 'hook_nodes', we could have a single class HookNode which has an attribute (e.g. 'checksum') which specifies what it does.